src/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ if (USE_CUDA)
     $<$<COMPILE_LANGUAGE:CUDA>:--expt-extended-lambda>
     $<$<COMPILE_LANGUAGE:CUDA>:--expt-relaxed-constexpr>
     $<$<COMPILE_LANGUAGE:CUDA>:-lineinfo>
-    $<$<COMPILE_LANGUAGE:CUDA>:--std=c++11>
+    $<$<AND:$<NOT:$<CXX_COMPILER_ID:MSVC>>,$<COMPILE_LANGUAGE:CUDA>>:--std=c++11>
```

Comment:
Is MSVC getting the right std version?

Reply:
MSVC doesn't have a flag for C++11. It only has `/std` flag for C++14 and C++17: https://stackoverflow.com/questions/54239927/does-cl-option-in-windows-makefile-will-accepts-std-c11-flag

Reply:
Warning from MSVC:
```
CUDACOMPILE : nvcc warning : The -std=c++11 flag is not supported with the configured host compiler. Flag will be ignored.
```